zomes/chat/src/entries/channel/handlers.rs

```diff
@@ -36,7 +36,7 @@ pub(crate) fn create_channel(channel_input: ChannelInput) -> ChatResult<ChannelD
     create_link(
         path.path_entry_hash()?,
         info_hash,
-        HdkLinkType::Paths,
+        HdkLinkType::Any,
```

Comment:
This is that link at the channel level I was talking about which conflicts with path links. I tried switching the link type so that `path.children()` wouldn't return it, but it didn't help. You could revert this change. I'm not sure what HdkLinkType means but I'm guessing that `Any` is the correct choice here anyway